src/gui/DatabaseWidget.cpp

```diff
@@ -1441,6 +1449,11 @@ void DatabaseWidget::reloadDatabaseFile()
         }
     }
 
+    // Lock out interactions
+    m_entryView->setDisabled(true);
+    m_groupView->setDisabled(true);
+    QApplication::processEvents();
```

Comment:
The clean way would be to actually to save in the background and queue any changes done in the meantime.

Reply:
That's a 2.6.0 change